tests/test_index.py

```diff
@@ -353,6 +353,58 @@ def test_add_embedding_index_if_exists(
         # sanity check persistence
         reload_tester.run_reload_test()
 
+    def test_update_img(
+        self,
+        img_tbl: pxt.Table,
+        test_tbl: pxt.Table,
+        clip_embed: func.Function,
+        e5_embed: func.Function,
+        all_mpnet_embed: func.Function,
+        reload_tester: ReloadTester,
+    ) -> None:
+        img_t = img_tbl
+        rows = list(img_t.select(img=img_t.img.fileurl, category=img_t.category, split=img_t.split).collect())
+        short_rows = rows[:5]
+        new_rows: list[dict[str, Any]] = []
+        for n, row in enumerate(short_rows):
+            row['pkey'] = n
+            new_rows.append(row)
+
+        # create table with fewer rows to speed up testing
+        schema = {'pkey': pxt.IntType(nullable=False), 'img': pxt.Image, 'category': pxt.String, 'split': pxt.String}
+        tbl_name = 'update_test'
+        img_t = pxt.create_table(tbl_name, schema, primary_key='pkey')
+        img_t.insert(new_rows)
+        print(img_t.head())
+
+        with pytest.raises(pxt.Error, match='computed column may not reference the errortype or errormsg'):
+            img_t.add_computed_column(emsg=img_t.img.errormsg)
+
+        with pytest.raises(pxt.Error, match='computed column may not reference the errortype or errormsg'):
+            img_t.add_computed_column(etype=img_t.img.errortype)
+
+        # Update the first row with a new image
```

Comment:
It would also be good to validate that computed columns that depend on the image column are also updated.